docs/user-guide/steerlm.rst

```diff
@@ -106,8 +111,56 @@ Download and Preprocess Data for Attribute Prediction Modeling
          --output-file=data/merge_val_reg.jsonl
 
 
-Train the Regression Reward Model on OASST+HelpSteer Data
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+If you are interested in replicating Reward Modelling training in HelpSteer2, please do the below instead.
```

Comment:
Line 43 fix capitalization (use title case for headings).

Train a SteerLM Model 


Line 51/51 fix capitalization  (use title case for headings).

Download the Llama 2 LLM Model 


Line 52/52 add ^ characters to align with length of heading.

Download the Llama 2 LLM Model 
^^^^^^^^^^^^^^^^^^^^^^^^^^


Line 78 revise sentence, fix punctuation.

To follow the HelpSteer2 and HelpSteer2-Preference line of works, you need to use the LLama 3 70B and LLama 3.1 70B Instruct models, respectively.


Line 80 revise sentence.

You need to obtain access to them, download them, and then convert them in a similar manner.


Line 82 remove numbered step, fix capitalization, fix spelling, and change heading to a sub-heading format (use ^ characters).

Download and Preprocess Data for SteerLM Regression Reward Modeling
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^


Line 88/93 fix punctuation.

#. Merge the two datasets for the train and val subset, respectively:


Line 114 revise sentence, fix spelling.

If you are interested in replicating Reward Modeling training in HelpSteer2, please follow the steps below instead.


Line 131 revise sentence, fix spelling.

If you're interested in replicating Reward Modeling training in HelpSteer2-Preference, please follow the steps below instead.


Line 161 remove numbered step, fix capitalization, fix spelling, and change heading to a sub-heading format (use ^ characters).

Train the Regression Reward Model on OASST+HelpSteer Data
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^


Line 112/165 revise sentence.

In this tutorial, you train the regression reward model for 800 steps. 


Line 194 revise sentence, fix spelling.

If you're interested in replicating Reward Modeling training in HelpSteer2, please follow the steps below instead.


Line 226 revise sentence, fix spelling.

If you're interested in replicating Reward Modeling training in HelpSteer2-Preference, please follow the steps below instead.

Reply:
fixed - thanks!